Tutorials/Sprite-Related-Tutorials/Getting Started With Sprites in Splashkit Tutorial - C#/Getting Started With Sprites in Splashkit Tutorial - CSharp.mdx

```diff
@@ -154,6 +225,36 @@ namespace learningSprites
       }
 }
 ```
+  </TabItem>
+  <TabItem label="C++">
+```c++
+#include "splashkit.h"
+
+int main()
+{
+    // Instantiation of Bitmap Object
+    bitmap HappySun = load_bitmap("Happy Sun", "sprite01.png");
+
+    // Instantiation of Sprite Object(s)
+    sprite HappySunSprite = create_sprite("Happy Sun", HappySun);
+
+    // Creation of Window
+    open_window("Learning Sprites!", 800, 800);
```

Comment:
I think you should change this to assign the window to a variable, like: `Window spriteWindow = open_window("Learning Sprites!", 800, 800);`